tests/rkt_gc_nspawn_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build host coreos src
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func getPodCgroups(shortUUID string) []string {
+	var podCgroups []string
+	walkCgroups := func(path string, info os.FileInfo, err error) error {
+		if info.Mode().IsDir() && strings.Contains(path, shortUUID) {
+			podCgroups = append(podCgroups, path)
+		}
+		return nil
+	}
+	_ = filepath.Walk("/sys/fs/cgroup", walkCgroups)
```

Comment:
We never return any error. I guess I can add a `if err != nil { return err }` inside the function and check it there, although I don't think it's really necessary.